documentation/add/07-sqw_redesign.md

```diff
@@ -225,11 +246,11 @@ Combine multiple experiment [data files](http://horace.isis.rl.ac.uk/Input_file_
 - Projection -- aggregate pixel data onto an M-dimensional surface (M &le; N). This should support projections onto planes and spherical shells (TBI) and spiral slices (TBI) as well as reorientations.
 - Symmetrization -- enhance signal-to-noise utilizing symmetries within the data
 
-The operations are performed on the pixel data and the image recalculated from that. If the pixel-data is not available (a DND object) the operations are performed directly on the image with reduced functionality but higher speed.
+The operations are performed on the pixel data and the image recalculated from that. If the pixel-data is not available (a `dnd` object) the operations are performed directly on the image with reduced functionality but higher speed.
```

Comment:
It's reduced accuracy, not reduced functionality, yes?

Reply:
probably reduced accuracy is better.